<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>370</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>535</width>
     <height>370</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#92AA83;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>11</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>6</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>4</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="pathOfSaveFolder">
        <property name="styleSheet">
         <string notr="true">	QLineEdit
{border-style:solid;
	border-width: 3px;
    border-radius: 10px;
    border-color: rgb(45, 0, 71);
    font: bold 12px;
    min-height:20px;
	max-width:400px;
    padding:3px;
background-color:#92AA83;
qproperty-alignment: AlignCenter;}
</string>
        </property>
        <property name="placeholderText">
         <string>Path of the folder where the converted file will be saved</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectFolderButton">
        <property name="styleSheet">
         <string notr="true">
QPushButton
{ font: bold 14px;
background-color:#FFBF00;
max-width:100px;
min-height:20px;

}
QPushButton:hover
{ font: bold 14px;
background-color:#FF7000;
}</string>
        </property>
        <property name="text">
         <string>Select folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>86</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"> font: bold 14px;
    min-height:20px;
	max-width:80px;
    padding:3px;

qproperty-alignment: AlignCenter;</string>
            </property>
            <property name="text">
             <string>IP Adress:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ipAdressInput">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>122</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">	QLineEdit
{border-style:solid;
	border-width: 3px;
    border-radius: 10px;
    border-color: rgb(45, 0, 71);
    font: bold 16px;
    min-height:20px;
	max-width:110px;
    padding:3px;
background-color:#92AA83;
qproperty-alignment: AlignCenter;}
QLineEdit:hover
{
    border-color: white;
}
QLineEdit:focus{
background-color:#92AA83;}</string>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>46</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
    font: bold 14px;
    min-height:20px;
	max-width:40px;
    padding:3px;
qproperty-alignment: AlignCenter;</string>
            </property>
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="portInput">
            <property name="styleSheet">
             <string notr="true">QLineEdit{	border-style:solid;
	border-width: 3px;
    border-radius: 10px;
    border-color: rgb(45, 0, 71);
    font: bold 16px;
    min-height:20px;
	max-width:50px;
    padding:3px;background-color:#92AA83;
qproperty-alignment: AlignCenter;}
QLineEdit:hover
{
    border-color: white;
background-color:#92AA83;
}
</string>
            </property>
            <property name="text">
             <string>9000</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>11</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="infoMessage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>177</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px;
qproperty-alignment: AlignCenter;</string>
            </property>
            <property name="text">
             <string>CAN'T CONNECT TO SERVER</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="pathOfFile">
        <property name="styleSheet">
         <string notr="true">	QLineEdit
{border-style:solid;
	border-width: 3px;
    border-radius: 10px;
    border-color: rgb(45, 0, 71);
    font: bold 12px;
    min-height:20px;
	max-width:400px;
    padding:3px;
background-color:#92AA83;
qproperty-alignment: AlignCenter;}
</string>
        </property>
        <property name="placeholderText">
         <string>Path of the file you wish to convert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectFileButton">
        <property name="styleSheet">
         <string notr="true">
QPushButton
{ font: bold 14px;
background-color:#FFBF00;
max-width:100px;
min-height:20px;

}
QPushButton:hover
{ font: bold 14px;
background-color:#FF7000;
}</string>
        </property>
        <property name="text">
         <string>Select file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">
    font: bold 14px;
    min-height:20px;
	max-width:50px;
    padding:3px;
	
qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selectTypeBox">
        <property name="styleSheet">
         <string notr="true">QComboBox{
background-color:#FFBF00;
font: bold;
selection-background-color:#FFBF00}
QComboBox::hover{
background-color:#FF7000;
}</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>asp-php</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>avi-flv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>avi-gif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>avi-mp4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bmp-gif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bmp-jpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bmp-png</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gif-avi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gif-flv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gif-mp4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jpg-bmp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jpg-gif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jpg-png</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mp4-flv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mp4-gif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pdf-html</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pdf-ps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pdf-txt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>png-bmp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>png-gif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>png-jpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ps-ascii</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ps-pdf</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tex-html</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>500</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>248</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="convertFileBtn">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton
{ font: bold 14px;
background-color:#FFBF00;
}
QPushButton:hover
{ font: bold 14px;
background-color:#FF7000;
}
QPushButton:disabled{

	background-color: rgb(186, 189, 182);

}</string>
        </property>
        <property name="text">
         <string>Convert</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
